docs/styles/styles.css

```diff
@@ -171,6 +171,10 @@
     margin-block: var(--rh-space-md, 8px);
   }
 
+  :where(h1, h2, h3, h4, h5, h6) {
+    margin-block-end: var(--rh-space-lg, 16px);
```

Comment:
@markcaron, this is a small edit that might be better to tackle in a different PR about docs spacing. If we have a heading + a heading, the space between them would be 32px.

Reply:
@marionnegp if it's ok, maybe we can push this out as it is, and then follow quickly after with #1610 adjusting heading + heading combinations. I don't think this will aversely affect docs.